cmd/jaeger/internal/extension/jaegerstorage/extension.go

```diff
@@ -104,6 +106,20 @@ func (s *storageExt) Start(_ context.Context, _ component.Host) error {
 		}
 		s.factories[storageName] = factory
 	}
+
+	for metricStorageName, cfg := range s.config.MetricBackends {
+		s.telset.Logger.Sugar().Infof("Initializing metrics storage '%s'", metricStorageName)
+		var metricsFactory storage.MetricsFactory
+		var err error = errors.New("empty configuration")
+		if cfg.Prometheus != nil {
+			metricsFactory = prometheus.NewFactory()
```

Comment:
```suggestion
			metricsFactory, err = prometheus.NewFactory(), nil
```

Reply:
Instead of `promethus.NewFactory` I have used `prometheus.NewFactoryWithConfig` which is similar to how other storages have implemented it .